taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -30,9 +30,10 @@ def add_variants(config, tasks):
         if build_type not in only_types:
             continue
 
-        for abi, apk_path in dep_task.attributes["apks"].items():
+        for abi, apk_metadata in dep_task.attributes["apks"].items():
             if abi not in only_abis:
                 continue
+            apk_path = apk_metadata["name"]
```

Comment:
Nit for this PR: Rename `name`

Reply:
I'll handle this in a followup PR. I don't remember what I did back in July so making this change in this PR adds some risk. I'd prefer to focus on getting this task up and running in production rather than fixing this kind of names.